### PR TITLE
Remove old motor legacy graph

### PR DESCRIPTION
**Pull request by McGiverGim**
When @ctzsnooze and me moved the motors to show the real command sent to motors, we left the legacy way to see if users prefer the old method. It seems nobody has complained, and I've seen some users confused about the legacy option. I think better to remove it. This reduces the amount of data the blackbox has to manage.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**
